README.md

```diff
@@ -12,53 +12,144 @@ A resource may classify its messages into groupings,
 and it may include additional data or metadata relating to them.
 It should be possible to represent a resource in a text-based human-friendly manner.
 
-### How to Contribute
-
-- **Issues** are for specific aspects of the resulting specification.
-  Discussions here should be focused, and stay on topic.
-  New issues must include a sufficient initial description to introduce themselves,
-  and may contain an exploration of possible solutions to them.
-- **Pull requests** help develop and define the eventual specification.
-  Accepting and merging a PR should not be taken as the final word on any topic,
-  but each change should be an improvement on the previous.
-  Each PR should only apply one change that may be squashed to the `main` branch.
-  Text content should use [semantic line breaks](https://sembr.org/).
-  A PR may close one or more issues, but this is not required.
-- **Discussions** are freeform, and may well refer to multiple issues or topics.
-  This is also the forum in which we should develop further our ways of working,
-  which may also include interactions and forums outside this repository,
-  such as occasional video calls.
-
-We welcome your participation and interest!
-
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The work on MessageFormat 2 has identified the following challenges
+that go beyond or arise from the new message format,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 syntax is naturally multi-line,
+  and multi-line values are not easy to use in many of the current resource formats.
```

Comment:
This reads a little ambiguously to me.

If you mean MF2 is better than current resource formats due to native multi-line,
```suggestion
- The MessageFormat 2 syntax is naturally multi-line,
  whereas multi-line values are not easy to use in many of the current resource formats.
```
or even
```suggestion
- The MessageFormat 2 syntax is naturally multi-line,
  whereas multi-line values are not well supported in many other resource formats.
```

Reply:
Ah, this is pointing at something else that needs better definition: MF2 is a _message_ format, and this effort is about defining a _resource_ format to support it. So comparing MF2 to "other resource formats" doesn't really make sense, because what comes out of this won't (necessarily) be a part of the MF2 spec, but a separate one.